Signal/src/ViewControllers/ConversationView/Cells/TypingIndicatorCell.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+@objc(OWSTypingIndicatorCell)
+public class TypingIndicatorCell: ConversationViewCell {
+
+    @objc
+    public static let cellReuseIdentifier = "TypingIndicatorCell"
+
+    @available(*, unavailable, message:"use other constructor instead.")
+    @objc
+    public required init(coder aDecoder: NSCoder) {
+        notImplemented()
+    }
+
+    private let kAvatarSize: CGFloat = 36
+    private let kAvatarHSpacing: CGFloat = 8
```

Comment:
These two constants are use mimic the appearance of the "real" message bubbles.  We use a single constant, but I didn't bother.